lambdas/cli.py

```diff
@@ -268,9 +268,8 @@ def validate_aip_via_lambda(
             "aip_s3_uri": aip_s3_uri,
             "verbose": verbose,
         },
-        timeout=900,  # long 15 min timeout for large AIPs
+        timeout=900,  # 15 min timeout (AWS Lambda maximum) for large AIPs
     )
-    response.raise_for_status()
```

Comment:
This was arguably the most important part.

Reply:
To make sure I understand, in the case where the AIP is missing, the Lambda was returning `generate_error_response`, and having `response.raise_for_status()` was hiding the content of that return value? 🤔